components/src/molecules/select/Select.stories.js

```diff
@@ -0,0 +1,55 @@
+import {storiesOf} from '@storybook/vue'
+
+import countries from '../../../mocks/countries.json'
+
+import ASelect from './Select.vue'
+
+storiesOf('Molecules/Select', module)
+  .addParameters({ info: true })
+  .add('Default', () => ({
+    components: { ASelect },
+    template: `
+      <a-select
+        id="field-id"
+        name="field-name"
+        :options="countryOptions"
+        selected="PL"
+      >
+        Label text
+      </a-select>
+    `,
+    computed: {
+      countryOptions() {
+        return countries.map((country) => {
```

Comment:
can we change mock instead of parsing this data?

Reply:
done